ui/components/MesheryPlayComponent.js

```diff
@@ -13,50 +11,40 @@ import { bindActionCreators } from "redux";
 import { setAdapter } from "../lib/store";
 
 const styles = (theme) => ({
-  icon : { fontSize : 20, },
-  playRoot : { padding : theme.spacing(0),
-    marginBottom : theme.spacing(2), },
-  buttons : { display : "flex",
```

Comment:
Also i think there are also some problems in some files like this as you can see playroot and margin bottom are not vertically aligned in the master branch as they should after formatting.